subgraph_predictor.py

```diff
@@ -15,14 +15,18 @@
 import kge.model
 import torch.nn.functional as F
 import nanopq
+import os
 
 from util import timer
 
+save_dir = '/var/scratch/dvs254/OpenKE-results/'
```

Comment:
Try not to use global variables to store directory paths and add an input parameter instead.